topics/metabolomics/tutorials/msPurity_training/tutorial.md

```diff
@@ -0,0 +1,1006 @@
+{: .text-justify}
+
+To analyze these data, we will start by following a light version of the [LC-MS workflow](http://workflow4metabolomics.org/the-lc-ms-workflow), developed by the [Wokflow4metabolomics group](http://workflow4metabolomics.org/) ({% cite Giacomoni2014 %}, {% cite Guitton2017 %}), then we will be able to complete the MS/MS workflow developed by **msPurity** authors ({% cite Lawson2017%}).
+{: .text-justify}
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Preprocessing with XCMS
+
+The first step of the workflow is the pre-processing of the raw data with **XCMS** ({% cite Smith2006 %}).
+{: .text-justify}
+
+**XCMS** is a free and open source software dedicated to pre-processing of any type of mass spectrometry acquisition files from low to high resolution, including FT-MS data coupled with different kind of chromatography (liquid or gas). This software is used worldwide by a huge community of specialists in metabolomics using mass spectrometry methods.
+{: .text-justify}
+
+This software is based on different algorithms that have been published, and is provided and maintained using R software.
+{: .text-justify}
+
+**MSnbase readMSData** {% icon tool %} function, prior to **XCMS**, is able to read files with open format as `mzXML`, `mzML`, `mzData` and `netCDF`, which are independent of the constructors' formats. The **XCMS** package itself is composed of R functions able to extract, filter, align and fill gap, with the possibility to annotate isotopes, adducts and fragments (using the R package CAMERA, {% cite CAMERA %}). This set of functions gives modularity, and thus is particularly well adapted to define workflows, one of the key points of Galaxy.
+{: .text-justify}
+
+First step of this tutorial is to download the data test. As describe in the introduction, we will use data from {% cite VanDerHooft2016 %}. We will only process on a subset of these data. So, you can **import your files directly in Galaxy by using the following URLs below** or download files into your computer (then upload them on Galaxy) : 
+{: .text-justify}
+
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3244991.svg)](https://doi.org/10.5281/zenodo.3244991)
```

Comment:
Have to update all zenodo links because I have to learn how to add my datatest on zenodo :)